src/postgres/src/backend/storage/ipc/procarray.c

```diff
@@ -2794,6 +2810,39 @@ MinimumActiveBackends(int min)
 	return count >= min;
 }
 
+void insertNode(PgProcAuhNode **head, PGProcAUHEntryList data) {
```

Comment:
This can be 

`void insertNode(PgProcAuhNode **head, PGProcAUHEntryList data) {`

and you can use `insertNode(head, data)`